src/components/featured-article/_macro.njk

```diff
@@ -0,0 +1,111 @@
+{% from "components/list/_macro.njk" import onsList %}
+{% from "components/chart/_macro.njk" import onsChart %}
+{% from "components/image/_macro.njk" import onsImage %}
+
+{% macro onsFeaturedArticle(params) %}
+    {% if params.headingLevel and params.headingLevel >= 1 and params.headingLevel <= 4 %}
+        {% set headingLevel = params.headingLevel %}
+    {% else %}
+        {% set headingLevel = 2 %}
+    {% endif %}
```

Comment:
Does this need to be limited to 1-4? We could simplify this whole if statement into one line e.g.
`{% set headingLevel = params.headingLevel | default(2) | string %}`